packages/embark/src/lib/modules/api/server.ts

```diff
@@ -112,47 +108,76 @@ export default class Server {
-              let timeLeft = ${reloadSeconds};
-              const span = document.querySelector("#timer");
-              setInterval(() => {
-                if (timeLeft >= 1) { timeLeft -= 1; }
-                span.innerText = \`\${timeLeft}\`;
-              }, 1000);
-            </script>
-          ` : ""}
         </body>
       </html>
     `.trim().split("\n").map((str) => str.trim()).filter((str) => str).join("\n"));
   }
 
   private makePage404(reloadSeconds: number, envReport: string, inside: string, notice: string) {
-    return this.makePage(reloadSeconds, `
+    return this.makePage(`
       ${envReport}
       <p>missing build for package <code>embark-ui</code> ${inside}</p>
       ${notice}
+      ${this.isInsideMonorepo ? `
+        <p>this page will automatically reload
+          in <span id="timer">${reloadSeconds}</span> seconds</p>
+        <script>
+          let timeLeft = ${reloadSeconds};
+          const span = document.querySelector("#timer");
+          const timer = window.setInterval(() => {
+            if (timeLeft >= 1) {
+              timeLeft -= 1;
+              span.innerText = \`\${timeLeft}\`;
+            }
+            if (!timeLeft) {
+              window.clearInterval(timer);
+              window.location.reload(true);
+            }
+          }, 1000);
+        </script>
+      ` : ""}
     `);
   }
 
-  private makePageEConnError(reloadSeconds: number, waitingFor: string) {
-    return this.makePage(reloadSeconds, `
-      <p><code>lib/modules/api/server</code> inside the monorepo at
-        <code>${path.join(this.monorepoRootDir, "packages/embark")}</code> is
-        waiting for the Create React App development server of package
-        <code>embark-ui</code> to ${waitingFor} at
-        <code>localhost:55555</code></p>
-      ${waitingFor === "become available" ? `
-        <p>please run either:</p>
-        <p><code>cd ${this.monorepoRootDir} && yarn start</code><br />
-          or<br />
-          <code>cd ${path.join(this.monorepoRootDir, "packages/embark-ui")}
-            && yarn start</code></p>
+  private makePage503(redirectSeconds: number) {
+    return this.makePage(`
+      <p><code>lib/modules/api/server</code> is inside the monorepo at
+        <code>${path.join(this.monorepoRootDir, "packages/embark")}</code></p>
+      <p>to access <code>embark-ui</code> in development use port
+        <code>3000</code></p>
+      <p>if you haven't already, please run either:</p>
+      <p><code>cd ${this.monorepoRootDir} && yarn start</code><br />
+        or<br />
+        <code>cd ${path.join(this.monorepoRootDir, "packages/embark-ui")} &&
+          yarn start</code></p>
         <p>to instead use a static build from the monorepo, restart embark with:
           <code>EMBARK_UI_STATIC=t embark run</code></p>
-      ` : ""}
+      <p>this page will automatically redirect to <a id="redirect" href="">
+        <span></span></a> in <span id="timer">${redirectSeconds}</span>
```

Comment:
The <span>  inside the <a> is useless as it's overwritten

Reply:
Updated, and I also made a change so that loading from `127.0.0.1:55555` works just as well as loading from `localhost:55555`.